src/index.css

```diff
@@ -56,4 +56,7 @@
   --primary-600: rgba(127, 86, 217, 1);
   --shadow-sm: 0px 1px 2px 0px rgba(16, 24, 40, 0.06), 0px 1px 3px 0px rgba(16, 24, 40, 0.1);
   --white: rgba(255,255,255,1);
```

Comment:
it's fine to get rid of these styles and keep the stylesheet clean if you don't need them, instead of commenting them out. If you need it later down the line, you can always check a previous commit to get it back.

Reply:
ok